packages/kbn-discover-utils/src/utils/format_hit.ts

```diff
@@ -79,7 +78,7 @@ export function formatHit(
       : [
           ...pairs.slice(0, maxEntries),
           [
-            i18n.translate('discover.utils.formatHit.moreFields', {
+            i18n.translate('discover-utils.formatHit.moreFields', {
```

Comment:
We could update `.i18nrc.json` to keep `discover` namespace https://github.com/elastic/kibana/blob/main/.i18nrc.json#L24

```
"discover": ["src/plugins/discover", "packages/kbn-discover-utils"],
```

Or lets change `discover-utils` to `discoverUtils` for consistency with others.

Reply:
:+1: Fixed in eacf973893af2009840cce5913f1d647d83bcd38.